Modules/xASL_module_ASL.m

```diff
@@ -472,21 +472,21 @@
         xASL_wrp_Quantify(x, x.P.Pop_Path_PWI4D_used);
         % Quantification in native space:
         xASL_wrp_Quantify(x, x.P.Path_PWI4D_used, x.P.Path_CBF, x.P.Path_rM0, x.P.Path_SliceGradient);
-    end
-    
-	% allow 4D quantification as well, storing CBF4D. This is currently not implemented for multi-PLD/multi-TE (BASIL/FABBER)
-	if x.modules.asl.SaveCBF4D
-		if size(xASL_io_Nifti2Im(x.P.Path_PWI4D_used), 4) == 1
-			warning('x.modules.asl.SaveCBF4D was requested but only one volume exists, skipping');
-		else
-			fprintf('%s\n','Quantifying CBF4D in native space');
-			xASL_wrp_Quantify(x, x.P.Path_PWI4D_used, x.P.Path_CBF4D, x.P.Path_rM0, x.P.Path_SliceGradient, true);
 
-			fprintf('%s\n','Quantifying CBF4D in standard space');
-			xASL_wrp_Quantify(x, x.P.Pop_Path_PWI4D_used, x.P.Pop_Path_qCBF4D, [], [], true);
+		% allow 4D quantification as well, storing CBF4D. This is currently not implemented for multi-PLD/multi-TE (BASIL/FABBER)
```

Comment:
We don't have to change this yet so might want to revert this.
But let's see.
Also: we now do this in both native & standard space.

Reply:
Why reverting? This was there already for a very long time...

Reply:
I meant that you are now defaulting to calculating CBF4D, we can revert this defaulting, and only start defaulting this when we indeed find out that this is useful.